### ISSUE TITLE
Refactor: Missing alt attributes on Issue Cover Images

### ISSUE THREAD

**Issue by Shailee6464** [Kritika75/TheCawnporeMag.github.io#237]
**Issue:**
Currently, the <img> tags for the magazine issue covers on the issue.html page are missing alt attributes. This impacts accessibility and search engine optimization (SEO).

**Affected File:** issue.html

**Proposed Solution:**
Add a descriptive alt attribute to each <img> tag within the .issuecover divs.

<img width="1161" height="693" alt="Image" src="https://github.com/user-attachments/assets/6444f368-fc0c-4301-8282-3572232a4938" />

I would like to contribute on this under GSSoC'25. Kindly assign me this.
